clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6421,8 +6443,8 @@ void OffloadBundler::ConstructJobMultipleOutputs(
   // contain bundled objects). We will perform partial linking against the
   // object and specific offload target archives which will be sent to the
   // unbundler to produce a list of target objects.
-  if (Input.getType() == types::TY_Object &&
-      TCArgs.hasArg(options::OPT_foffload_static_lib_EQ)) {
+  bool StaticLib = TCArgs.hasArg(options::OPT_foffload_static_lib_EQ);
+  if (Input.getType() == types::TY_Object && StaticLib) {
```

Comment:
StaticLib seems to be still used just once - unnecessary change?

Reply:
It was necessary, until I made some changes.  I'll clean out.